app/src/main/java/com/github/orkest/ui/FollowListViewModel.kt

```diff
@@ -1,29 +1,40 @@
 package com.github.orkest.ui
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.github.orkest.data.User
 import com.github.orkest.domain.FireStoreDatabaseAPI
+import java.util.concurrent.CompletableFuture
 
-class FollowListViewModel(val username: String, val isFollowersList: Boolean) {
+open class FollowListViewModel(val username: String, val isFollowersList: Boolean) {
 
     private val dbAPI = FireStoreDatabaseAPI()
```

Comment:
the FirestoreDataBaseAPI is now static so you don't need to recreate an instance for it each time you can directly use its methods with FirestoreDataBaseAPI.fetch..